code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -109,18 +115,26 @@ var/global/mulebot_count = 0
 	return
 
 /mob/living/simple_animal/bot/mulebot/emag_act(mob/user)
-	locked = !locked
-	user << "<span class='notice'>You [locked ? "lock" : "unlock"] the mulebot's controls!</span>"
+	if (emagged < 1)
+		emagged = 1
+	if (!open)
+		locked = !locked
+		user << "<span class='notice'>You [locked ? "lock" : "unlock"] the [src]'s controls!</span>"
+	else if (emagged < 2)
+		emagged = 2
+		if(!wires.is_cut(WIRE_AVOIDANCE))
+			wires.cut(WIRE_AVOIDANCE)
+		user << "<span class='warning'>You awaken [src]'s thirst for blood!</span>"
+		src << "<span class='userdanger'>(#$*#$^^( OVERRIDE DETECTED</span>"
+		add_logs(user, src, "emagged")

```

Comment:
I'm really not sure whether we should be adding more uses to the emag (already the best traitor item by far). I'd rather just leave it up to wires than doing this.